src/views/AuthenticationWrapper/DevSecurity.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { ReactEventHandler, useEffect, useRef, useState } from 'react';
+import { OktaContext } from '@okta/okta-react';
+
+const storageKey = 'dev-user-config';
+
+const initialAuthState = {
+  isAuthenticated: false,
+  isPending: false,
+  name: '',
+  euaId: '',
+  groups: [] as string[]
```

Comment:
I recommend this being an object with the groups as keys. This way you can easily have the groups checked/unchecked by default, not need to use refs/sets, and add other complexities. I regret using arrays to handle the collaborators on the system intake.

So far example, groups would look like:
```
const initialAuthState = {
  ...,
  groups: {
    EASI_D_GOVTEAM: false,
    EASI_P_GOVTEAM: false
  }
}
```